frontend/src/app/utils/user.ts

```diff
@@ -15,3 +15,16 @@ export const formatFullName = <T extends RequiredUserFields>(user: T) => {
   result += user.suffix ? `, ${user.suffix}` : "";
   return result;
 };
+
+export const formatRole = (role: String) => {
+  let result = "";
+  role = role.toLowerCase();
+  if (role.includes("admin")) {
```

Comment:
@Rebecca-LM this is what I changed the role descriptions to, so that they match the options in manage users.

The only place they appear is in the user button on the header:
![Screen Shot 2021-08-31 at 3 11 53 PM](https://user-images.githubusercontent.com/80282552/131582462-54bc86c5-8cff-426f-85ea-59b83482134c.png)

That `Role - Admin user` would now read `Role - Admin`